Projects/ChangeTier/Sources/Views/ChangeTierLandingScreen.swift

```diff
@@ -0,0 +1,298 @@
+                        VStack(alignment: .leading, spacing: 0) {
+                            hText(vm.displayName ?? "")
+                            hText(vm.exposureName ?? "")
+                                .foregroundColor(hTextColor.Opaque.secondary)
+                        }
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                    }
+                }
+
+                VStack(spacing: .padding4) {
+                    if !vm.canEditTier {
+                        hSection {
+                            VStack(alignment: .leading, spacing: .padding4) {
+                                hFloatingField(
+                                    value: vm.selectedTier?.name ?? "",
+                                    placeholder: L10n.tierFlowCoverageLabel
+                                ) {}
+                                .hFieldLockedState
+                                .hFieldTrailingView {
+                                    Image(uiImage: hCoreUIAssets.lock.image)
+                                        .foregroundColor(hTextColor.Opaque.secondary)
+                                }
+                                hText(L10n.tierFlowLockedInfoDescription, style: .label)
+                                    .foregroundColor(hTextColor.Translucent.secondary)
+                                    .padding(.leading, .padding16)
+                            }
+                        }
+                        .padding(.bottom, 8)
+                    } else {
+                        DropdownView(
+                            value: vm.selectedTier?.name ?? "",
+                            placeHolder: vm.selectedTier != nil
+                                ? L10n.tierFlowCoverageLabel : L10n.tierFlowCoveragePlaceholder
+                        ) {
+                            changeTierNavigationVm.isEditTierPresented = true
+                        }
+                    }
+
+                    DropdownView(
+                        value: vm.selectedDeductible?.deductibleAmount?.formattedAmount ?? "",
+                        placeHolder: vm.selectedDeductible != nil
+                            ? L10n.tierFlowDeductibleLabel : L10n.tierFlowDeductiblePlaceholder
+                    ) {
+                        changeTierNavigationVm.isEditDeductiblePresented = true
+                    }
+                    .disabled(vm.selectedTier == nil)
+                }
+                .hFieldSize(.small)
+                .hWithTransparentColor
+                .hWithoutHorizontalPadding
+
+                hRow {
+                    HStack(alignment: .top) {
+                        hText(L10n.tierFlowTotal)
+                        Spacer()
+                        VStack(alignment: .trailing, spacing: 0) {
+                            hText(
+                                vm.newPremium?.formattedAmountPerMonth ?? vm.currentPremium?.formattedAmountPerMonth
```

Comment:
Can we rework this to be from the viewModel?
So like 
```
Published var newPrice: String?
Published var oldPrice: String?
```
And update it in the viewModel
so we end up with:
```
VStack(alignment: .trailing, spacing: 0) {
                            hText(
                                vm.newPrice
                            )

                            if let oldPrice = vm.oldPrice {
                                hText(
                                    L10n.tierFlowPreviousPrice(oldPrice),
                                    style: .label
                                )
                                .foregroundColor(hTextColor.Opaque.secondary)
                            }
                        }
```

Reply:
Fixed in next PR https://github.com/HedvigInsurance/ugglan/pull/1525/files#diff-a3ed6027db3cc056ee408b56c8822cf95722fcba47d286add1443456db550ab8